site/content/entry/yuzu-progress-report-may-2022/index.md

```diff
@@ -0,0 +1,235 @@
++++
+date = "2022-06-10T05:00:00-03:00"
+title = "Progress Report May 2022"
+author = "GoldenX86"
+coauthor = ""
+forum = 0
++++
+
+Greetings yuz-ers. While this monthly report will not be as extensive as others, we have several critical changes to discuss. Roll the tape!
```

Comment:
Remove " not as extensive" and instead say something like:

"This time around, we're covering many small and incremental improvements to yuzu. Rest assured, we also have some *major* rewrites and improvements in the works, and we'll touch on those near the end. Roll the tape!"